toolchain/templates/delta.mako

```diff
@@ -5,7 +5,7 @@
 % if engine == 'batch':
 #SBATCH --nodes=${nodes}
 #SBATCH --ntasks-per-node=${tasks_per_node}
-#SBATCH --cpus-per-task=1
+#SBATCH --cpus-per-task=16
```

Comment:
are you sure about this change? @wilfonba can comment

Reply:
Yes, Ben recommended this change at APS DFD. I can changed it back.